.github/workflows/tests.yml

```
@@ -49,41 +49,31 @@ jobs:
-      if: ${{ matrix.c_compiler == 'clang' }}
-      run: >
-        cmake -B ${{ steps.strings.outputs.build-output-dir }}
-        -DCPM_SOURCE_CACHE=~/cpm-cache
-        -DCMAKE_CXX_COMPILER=clang++
-        -DCMAKE_C_COMPILER=clang
-        -DCMAKE_BUILD_TYPE=${{ matrix.build_type }}
-        -DMR_MATH_ENABLE_TESTING=ON
-        -S ${{ github.workspace }}
+    - name: Install Conan
+      uses: turtlebrowser/get-conan@main
+ 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)